pkgs/development/python-modules/duckduckgo-search/default.nix

```
@@ -2,28 +2,38 @@
 , fetchFromGitHub
 , lib
 , setuptools
-, requests
+, aiofiles
 , click
+, h2
+, httpx
+, lxml
+, requests
+, socksio
 }:
 
 buildPythonPackage rec {
   pname = "duckduckgo-search";
-  version = "2.8.5";
+  version = "3.8.4";
 
   src = fetchFromGitHub {
     owner = "deedy5";
     repo = "duckduckgo_search";
     rev = "v${version}";
-    hash = "sha256-UXh3+kBfkylt5CIXbYTa/vniEETUvh4steUrUg5MqYU=";
+    hash = "sha256-FdxmIDmbQREQd0OnL36PjOrtfCEG7/nab25g+EWoP8M=";
   };
 
   format = "pyproject";
 
   nativeBuildInputs = [ setuptools ];
 
   propagatedBuildInputs = [
-    requests
+    aiofiles
     click
+    h2
+    httpx
+    lxml
+    requests
+    socksio
   ];
```

```suggestion
    httpx
    lxml
  ] ++ httpx.optional-dependencies.https2
    ++ httpx.optional-dependencies.socks
    ++ httpx.optional-dependencies.brotli;
```
Would you please add dependencies with using `optional-dependencies`?